src/backend/utils/rmFolderRecursive.ts

```diff
@@ -0,0 +1,42 @@
+import fs from 'fs'
+
+export function deleteFolderRecursive(path: string) {

```

Comment:
why not using rmSync with recursive flag?

```ts
fs.rmSync(dir, { recursive: true, force: true });
```

Reply:
doesn't work unfortunately. only works if the folder is already empty

tried all the solutions from https://stackoverflow.com/questions/18052762/remove-directory-which-is-not-empty and this is the only one that worked